integrations/oracledb/ops_agent_metadata.yaml

```diff
@@ -1,7 +1,7 @@
 platforms:
 - type: GCE
-  launch_stage: GA
-  install_documentation_url: https://cloud.google.com/stackdriver/docs/solutions/agents/ops-agent/third-party/oracledb
+  launch_stage: HIDDEN
+  launch_date: '2022-11-17'
```

Comment:
@rafaelwestphal This is going to remove the oracle integration from the integration portal. I assume this was unintentional?

Reply:
This is due to https://github.com/GoogleCloudPlatform/ops-agent/pull/924 was never merged in.

Reply:
Thanks for looking into that, do we need to re-run the script to generate this PR?

Reply:
yes, i just triggered it ->https://github.com/GoogleCloudPlatform/monitoring-dashboard-samples/pull/364

Reply:
Thank you!